OpenKeychain/src/main/java/org/sufficientlysecure/keychain/remote/ui/RemoteSelectPubKeyActivity.java

```diff
@@ -103,7 +103,7 @@ protected void onCreate(Bundle savedInstanceState) {
                     public void onClick(View v) {
                         // add key ids to params Bundle for new request
                         Intent resultData = extras.getParcelable(EXTRA_DATA);
-                        resultData.putExtra(OpenPgpApi.EXTRA_KEY_IDS,
+                        resultData.putExtra(OpenPgpApi.EXTRA_KEY_IDS_SELECTED,
```

Comment:
why do we need an additional extra? What about merging the additional key ids with the existing given ``EXTRA_KEY_IDS`` before returning ``EXTRA_KEY_IDS``?